.github/ISSUE_TEMPLATE/release.md

```diff
@@ -131,6 +137,11 @@ in the infrastructure repository. All of these tests can be run in parallel.
   git push origin $tag
   ```
 
+Go to [BuildKite > Tests](https://buildkite.com/materialize/tests) and verify that a job called
```

Comment:
I don't follow this. I don't think the "Tests" pipeline has a "release" job. Do you mean something in the "Deploy" pipeline?

Reply:
There is a Release job under Tests: https://buildkite.com/materialize/tests/builds/10716. Perhaps my terminology is wrong?

Reply:
Ohh, that's just the title of the commit that triggered the build. "Jobs" in Buildkite terminology are all the individual steps in the pipeline, like "cargo test" or "metabase demo." A perfectly precise description is probably something like this:

> Before you can verify the Debian package, you will need to wait for the [Buildkite "Deploy" build](https://buildkite.com/materialize/deploy) for the tag's commit to complete. Note that the Deploy pipeline is not triggered unless the [Test pipeline](https://buildkite.com/materialize/tests) for the tag's commit completes successfully. The whole process can take about 15m to complete.

Feel free to cut that down to size, since that's a mouthful!

Reply:
Cool, thanks. I'll update accordingly.